applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -385,7 +383,7 @@ int ble_vcs_volume_mute(void)
 #elif (CONFIG_BT_VCS)
 	return bt_vcs_mute(vcs);
 #endif /* (CONFIG_BT_VCS_CLIENT) */
-	return -ENXIO;
+	return hw_codec_volume_mute();
 }
 
 int ble_vcs_volume_unmute(void)
```

Comment:
Should hw_codec_volume_unmute() be added to ble_vcs_volume_unmute()?

Reply:
Added this feature